extensions/scripting/lua-bindings/manual/network/lua_xml_http_request.cpp

```diff
@@ -219,7 +219,7 @@ void LuaMinXmlHttpRequest::_sendRequest()
 
             if (!response->isSucceed())
             {
-                CCLOG("Response failed, statusCode: %s", response->getResponseCode());
+                CCLOG("Response failed, statusCode: %ld", response->getResponseCode());
```

Comment:
```suggestion
                CCLOG("Response failed, statusCode: %d", response->getResponseCode());
```

Reply:
And the `long statusCode = response->getResponseCode();` should be change to `int statusCode = response->getResponseCode();`